pom.xml

```diff
@@ -23,6 +23,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <source>8</source>
 
+        <mvn.version>3.0.0</mvn.version>
```

Comment:
Should this be 3.1.0?

Reply:
The old prerequisites section said 3.0.0:
```
<prerequisites>
    <maven>3.0.0</maven>
</prerequisites>
```

Does this plugin actually require 3.1.0?

Reply:
I realise now that this branch was out-of-date - so yes, I guess this should be 3.1.0 now.